libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp

```diff
@@ -1120,5 +1128,31 @@ bool AP_ExternalAHRS_InertialLabs::get_variances(float &velVar, float &posVar, f
     return true;
 }
 
+bool AP_ExternalAHRS_InertialLabs::write_bytes(const char *bytes, uint8_t len)
+{
+    return uart->write(reinterpret_cast<const uint8_t *>(bytes), len);
+}
+
+void AP_ExternalAHRS_InertialLabs::set_data_sending_state(bool enabled)
+{
+    if (enabled)
```

Comment:
nitpick: bracket should be on the same line as the if

Reply:
Done